.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-added-large-files
+
+  - repo: https://github.com/adrienverge/yamllint
+    rev: v1.26.3
+    hooks:
+      - id: yamllint
+        args: [-c=.yamllint]
+
+  - repo: https://github.com/ansible-community/ansible-lint.git
+    rev: v6.10.2
+    hooks:
+      - id: ansible-lint
+        pass_filenames: false
+        entry: ansible-lint --force-color -v
+        additional_dependencies:
+          - .[community]
```

Comment:
You also shouldn't need this if you're only using Ansible builtin modules in your tasks/handlers. I have this enabled in my project as I'm using builtin + community modules.

Reply:
rm-ed both mentioned blocks. `pre-commit` runs fine still, when I run it locally.
Yes, it was just copy-pasted without specific needs.

I commit the changed file in a minute.